Maddori.Apple/Maddori.Apple/Global/UIComponent/LabelButtonView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Label+ButtonView.swift
+//  Maddori.Apple
+//
+//  Created by 이성호 on 2022/10/22.
+//
+
+import UIKit
+
+import SnapKit
+
+class LabelButtonView: UIView {
+    
+    var buttonAction: (() -> ())?
+    
+    var subText: String = "" {
+        didSet {
+            subLabel.text = subText
+        }
+    }
+    
+    var subButtonText: String = "" {
+        didSet {
+            subbutton.setTitle(subButtonText, for: .normal)
+        }
+    }
+    
+    // MARK: - property
+    private lazy var subbutton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle(subButtonText, for: .normal)
+        button.titleLabel?.font = .body2
+        let action = UIAction { [weak self] _ in
+            self?.buttonAction?()
+        }
+        button.addAction(action, for: .touchUpInside)
+        return button
+    }()
+    
+    private lazy var subLabel: UILabel = {
+        let label = UILabel()
+        label.text = subText
+        label.font = .body2
+        label.textColor = .gray400
+        return label
+    }()
+    
+    // MARK: - init
+    override init(frame: CGRect) {
```

Comment:
p4;

```suggestion
    // MARK: - init
    
    override init(frame: CGRect) {
```

요기도 한줄만.... 하하ㅏ하하하

Reply:
[0a0348b](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/32/commits/0a0348b172d455e730bdc2535fae0a1ac131597e)